image-viewer.py

```diff
@@ -56,72 +59,87 @@ def render(self):
 
         self.root.bind("<KeyPress>", self.onKeyPress)
         self.root.mainloop()
-        
+
     def create_menu(self):
         menubar = Menu(self.root)
         self.root.config(menu=menubar)
-        
+
         file_menu = Menu(menubar, tearoff=False)
-        file_menu.add_command(label='Open Folder', command=self.open_folder, underline=0)
-        file_menu.add_command(label='Refresh', command=self.refresh, underline=0)
-        file_menu.add_command(label='Exit', command=self.root.destroy, underline=1)
-        menubar.add_cascade(label='File', menu=file_menu, underline=0)
-    
+        file_menu.add_command(label="Open file(s)", command=self.open_files)
+        file_menu.add_command(label="Open Folder", command=self.open_folder, underline=0)
+        file_menu.add_command(label="Refresh", command=self.refresh, underline=0)
+        file_menu.add_command(label="Exit", command=self.root.destroy, underline=1)
+        menubar.add_cascade(label="File", menu=file_menu, underline=0)
+
+    def open_files(self):
+        files = filedialog.askopenfilenames(
+            filetypes=(
+                ("all files accepted", ["*.png", ".jpg", ".ico", ".gif", ".jpeg"]),
+                ("png files", "*.png"),
```

Comment:
A minor point that I'm not too fussed about, it's merely my preference, but I think it would be good if these matched how other applications listed their supported file type, like:
![image](https://user-images.githubusercontent.com/54030147/194771759-6b3fd3a9-d325-409a-bef6-65a5971f7ad6.png)

(ignore the file types that we don't have listed here, like HEIC and WEBP)